.github/workflows/sync-status-check.yml

```
@@ -27,24 +27,26 @@ jobs:
           $scriptUrl = "${{ vars.SCRIPT_BASE_URL }}/${{ vars.CICD_SCRIPTS_VERSION }}/${{ vars.SCRIPT_RELATIVE_DIR_PATH}}/sync-bot-status-check.ts";
           $prNumber = "${{ github.event.number }}";
 
-          Write-Host "::notice::Project Name: ${{ vars.PROJECT_NAME }}";
-          Write-Host "::notice::PR Number: $prNumber";
-          Write-Host "::notice::Event Type: pr";
+          Write-Host "::notice:Project Name: ${{ vars.PROJECT_NAME }}";
+          Write-Host "::notice:PR Number: $prNumber";
+          Write-Host "::notice:Event Type: pr";
 
           if ($manuallyExecuted -and $prNumber -eq "0") {
-            Write-Host "::error::The issue or PR number must be a value greater than 0.";
+            Write-Host "::notice:The issue or PR number must be a value greater than 0.";
```

Refactor the message type from `notice` to `error`.

```suggestion
            Write-Host "::error::The issue or PR number must be a value greater than 0.";
```